TLM/TLM/Manager/Impl/JunctionRestrictionsManager.cs

```diff
@@ -26,25 +26,25 @@ public class JunctionRestrictionsManager
         public static JunctionRestrictionsManager Instance { get; } =
             new JunctionRestrictionsManager();
 
-        private readonly SegmentFlags[] invalidSegmentFlags;
+        private readonly SegmentJunctionRestrictions[] invalidSegmentRestrictions;
 
         /// <summary>
         /// Holds junction restrictions for each segment end
         /// </summary>
-        private readonly SegmentFlags[] segmentFlags_;
+        private readonly SegmentJunctionRestrictions[] segmentRestrictions;
```

Comment:
naming convention: private fields should end with a _
Did you not get a warning on your IDE?

Reply:
oh you already merged. I am too late. 
Nevermind its just a small issue.

Reply:
No, I'm not seeing a warning.

Reply:
I also don't get warning about private fields should end with `_`. I'm using VS 2019 (soon I'll be updating to VS 2022).

Reply:
tbh, it's kind of a weird convention that I've never even heard of in nearly 40 years of coding. 🤷‍♂️

*Starting* the name of a private field with `_` is a thing, though I usually just do that on members that should only be referenced through accessors and never directly.

Reply:
^ This

Reply:
It should end with _

Reply:
do you use the same naming convention for public/private fields?

Reply:
Yes.

Do you use the same for public and private methods? Why should fields be different?

Reply:
Can this discussion be moved to issue/discussion so it can be properly tracked rather than buried in an closed PR?

Reply:
@kvakvs did you create a page for naming convention?

Reply:
https://github.com/CitiesSkylinesMods/TMPE/wiki/Code-style-and-naming
but please check the project and linter settings if they don't match, its good to fix them for automatic enforcement